go.mod

```diff
@@ -5,9 +5,9 @@ go 1.15
 require (
 	github.com/cloudevents/sdk-go/v2 v2.4.1
 	github.com/prometheus/client_golang v1.11.0
-	github.com/redhat-cne/rest-api v0.0.0-20210628151454-c8f37f0a9c3e
-	github.com/redhat-cne/sdk-go v0.0.0-20210630032545-4072da5111ba
+	github.com/redhat-cne/rest-api v0.1.0
```

Comment:
you might have to use main branch commit.

Reply:
will update mod file once the sdk-go redfish-schema change is delivered.

Reply:
this needs to be fixed